muffnn/mlp/mlp_classifier.py

```diff
@@ -129,19 +129,23 @@ def _init_model_objective_fn(self, t):
         if self.multilabel_:
             cross_entropy = tf.nn.sigmoid_cross_entropy_with_logits(
                 logits=t, labels=tf.cast(self.input_targets_, np.float32))
+            cross_entropy = tf.multiply(cross_entropy, self._sample_weight)
```

Comment:
Does this broadcast correctly?

For multilabel, I think the shape of of cross_entropy is [n_examples, n_classes] and the weights are [n_examples]. You might need to add a trailing dimensions to get this to work?

Reply:
It does not broadcast correctly. The fact that the tests passed makes me worry how well we're testing the multi-label case.

Reply:
Agreed. Sam and I worked on those tests a lot but I am not going to claim they are bullet proof.